book.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Create the Book class
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] The book `add_rental` method should also contain the date parameter, some like so: 
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
Please kindly fix that.

Reply:
Well solved